### PR TITLE
Cdk support

### PR DESCRIPTION
**Pull request by jyriseiger**
Added first version of CDK support.

### CODE REVIEW

File: demo.tape
Comment (Maurits-de-Groot) [inline demo.tape:1]: Maybe we should NOT add this file?

Comment (jyriseiger) [reply demo.tape:1]: agree, removed it.

File: susscanner/cli.py
Comment (Maurits-de-Groot) [inline susscanner/cli.py:1]: Could we use subprocess.Popen over system.os?

File: README.md
Comment (Maurits-de-Groot) [inline README.md:1]: Under Prerequisites we should list cdk as optional pre-req for the cdk support

Comment (Maurits-de-Groot) [reply README.md:1]: Looks good, thanks for adding

**Review by Maurits-de-Groot — APPROVED** [review]

**Pull request merged**